tests/circuit_drawer/test_circuit_drawer.py

```diff
@@ -93,7 +93,7 @@ def to_layer(operation_list, num_wires):
     layer = [None] * num_wires
 
     for op in operation_list:
-        for wire in op.wires:
+        for wire in op.wires.tolist():
```

Comment:
how come `tolist()` is still needed here? :thinking:

Reply:
I was just too lazy to rewrite the entire tests from bottom up - we can just avoid this function in tests that use the non-consec wire feature for now? Will add a todo though, would that be ok?